src/config.rs

```diff
@@ -128,6 +139,18 @@ impl Context {
         self.get_config_int(key) != 0
     }
 
+    /// Gets configured "delete_server_after" value.
+    ///
+    /// `None` means never delete the message, `Some(0)` means delete
+    /// at once, `Some(x)` means delete after `x` seconds.
+    pub fn get_config_delete_server_after(&self) -> Option<i64> {
```

Comment:
Why not use an actual enum for this?  Having 0 as a special meaning isn't very nice I think

Reply:
Is it special? 0 means "after 0 seconds" here, which is the same as "at once".

Reply:
i would also not use an enum, it is already confusing enough - 1 in ffi is Some(0) in rust. maybe we should just use 1 second for "at once" - and use the same values for ffi and in rust. this way, the function get_config_delete_server_after() won't be needed at all.